posts/api_enabled_retail.md

```diff
@@ -0,0 +1,67 @@
+
+In today's world, customers are demanding a cohesive experience that this isolationist approach can hinder.
+
+## Customers Increasingly Expect a More Connected Experience
+
+Customers are interacting with retailers through an increasing array of touchpoints, from web, to mobile, and in-store, and perhaps third party marketplaces.
+
+An organisation that has adopted an API ready approach will find integrating new channels a much easier experience. If the e-commerce platform already exposes an API for accepting payments, half the job would already be done in allowing a new mobile application the same functionality. Likewise if the warehouse system already provides an API for stock levels, showing live stock data on the e-commerce storefront is largely solved.
+
+Centralising customer data in to a CRM system that can be accessed via APIs allows a customer to be more easily recognised across the retail touchpoints, and can enable the shopping experience to be tailored to them. A salesperson who can see a customers recent browsing habits and online purchases is much better able to make recommendations. Likewise, if the data feeds both ways, customers can be retargeted with relevant advertising online based on their offline behaviour.
+
+Many retailers already have significant maturity in delivering APIs
+
+## Existing API-enabled Retailers
+
+As noted, we observe many organisations, particularly lead by those with a strong online presence, adopting an API enabled strategy.
+
+Perhaps the most notable moves Jeff Bezos made at Amazon was to mandate that [every service must expose an API](https://apievangelist.com/2012/01/12/the-secret-to-amazons-success-internal-apis/) (and you'll be fired if you don't!). That mandate was responsible for sewing the seeds of Amazon Web Services, now itself a $12 billion a year business.
+
+On a smaller scale, startups are entering many industries and adopting or building modern, API enabled software to do so. Payment processor Stripe has significantly lowered the barrier to entry in processing online payments, in part by how easy they've made it to integrate with your application. Algolia offers a search platform bettering many existing enterprise offerings, enabling developers to be up and running within an hour.
+
+Offering APIs can open opportunities outside of the organisation too.
+
+## Retail APIs and Third Party Services
+
+Integrating with third party services easily can be a compelling benefit. Enabling commerce anywhere (think "shop the look" on Instagram), or exposing product catalogues to price comparison or aggregation services without the need for a standalone channel manager to power an integration.
+
+Opportunities can be opened to work with partner organisations with complimentary offerings, or to encourage third party developers to interface with your systems. Reportedly 60% of eBay's revenue is driven through API calls, and for Expedia the number is reportedly as high as 90%.
```

Comment:
Repeated reportedly in that last line - could also be a good place to slot a link in, with a reference to where those stats come from?